include/util/node_based_graph.hpp

```diff
@@ -51,14 +51,19 @@ struct NodeBasedEdgeData
     LaneDescriptionID lane_description_id;
     extractor::guidance::RoadClassification road_classification;
 
-    bool IsCompatibleTo(const NodeBasedEdgeData &other) const
+    bool IsCompatibleToExceptForName(const NodeBasedEdgeData &other) const
     {
-        return (name_id == other.name_id) && (reversed == other.reversed) &&
+        return (reversed == other.reversed) &&
                (roundabout == other.roundabout) && (startpoint == other.startpoint) &&
                (access_restricted == other.access_restricted) &&
                (travel_mode == other.travel_mode) &&
                (road_classification == other.road_classification);
     }
+
+    bool IsCompatibleTo(const NodeBasedEdgeData &other) const
```

Comment:
Maybe we can find better names here reflecting where each is used. I would imagine something like:

- `IsCompressableWith` (currently `IsCompatibleTo`) since it is used by the graph compressor
- `IsCompatibleTo` (currently `IsCompatibleToExceptForName`) since it is used in the guidance code